assets/src/scss/pages/_page.scss

```diff
@@ -3,11 +3,11 @@ div.page-template {
   max-width: 100%;
   margin-left: auto;
   margin-right: auto;
-  padding-left: 15px;
-  padding-right: 15px;
+  padding: 0 16px;
 
   @include medium-and-up {
     width: 720px;
+    padding: 0;
```

Comment:
Leaving the padding on bigger screen sizes was causing the content to be smaller than the expected width